sdk-tests/src/test/java/io/dapr/it/methodinvoke/http/GH_1314_IT.java

```diff
@@ -0,0 +1,87 @@
+package io.dapr.it.methodinvoke.http;
```

Comment:
Can we add a header to this file and change the name to something more intuitive like query_encoding_IT or query_param_encoding_IT or similar? We don't typically name files based on the GH issue bc future developers wont know what is in the test without an extra step of pulling up GH and looking up the issue, although I appreciate you trying to reference it that way.

Reply:
Done